specification/evaluation-context/evaluation-context.md

```diff
@@ -1,5 +1,41 @@
+
+### Fields
+
+##### Requirement 3.1
+
+> The `evaluation context` structure **MUST** define a required `targeting key` field of type string, identifying the subject of the flag evaluation.
+
+The targeting key uniquely identifies the subject (end-user, or client service) of a flag evaluation. Providers may use this as a unique identifier for fractional-flags, rules, or overrides targeting specific users.
+
+##### Requirement 3.2
+
+> The evaluation context **MUST** define the following optional fields: `email` (string), `first name` (string), `last name`(string), `name`(string), `client ip`(string), `tz`(string), `locale`(string), `date`(string | date), `environment`(string), `application`(string), and `version`(string).
+
+See the table below for details.
+
+| Name        | Type           | Purpose                                            | Example(s)                                                |
+| ----------- | -------------- | -------------------------------------------------- | --------------------------------------------------------- |
+| email       | string         | Subject RFC 5322 conformant email                  | "first.last@example.com"                                  |
+| first name  | string         | Subject given name                                 | "Jane"                                                    |
+| last name   | string         | Subject last name                                  | "Smith"                                                   |
+| name        | string         | Subject full name and titles and suffixes          | "Ms. Jane Smith"                                          |
+| client ip   | string         | Client IP address (IPv4 or IPv6)                   | "216.7.132.218","2001:0db8:85a3:0000:0000:8a2e:0370:7334" |
+| tz          | string         | Subject "zoneinfo" timezone                        | "Brazil/West","America/New_York"                          |
+| locale      | string         | Subject RFC 5646 locale                            | "pt-BR","en-US"                                           |
+| date        | date \| string | Date primitive or date in extended ISO 8601 format | "2022-05-10T01:53:29.823Z"                                |
+| environment | string         | Identifier for deployed environment                | "prod", "development"                                     |
+| application | string         | Application identifier                             | "fibonacci-api"                                           |
+| version     | string         | Application version                                | "1.3.1","v2-beta"                                         |
+
+\*NOTE: Field casing is not specified, and should be chosen in accordance with language idioms.
```

Comment:
I don't think we have to worry about the casing much since moving this data over the wire isn't really a concern... we just want to make sure it's reflected in the `evaluation context` structure in any implementation.